web/src/engine/websites/MangaLib_e2e.ts

```diff
@@ -1,24 +1,47 @@
 ﻿import { TestFixture } from '../../../test/WebsitesFixture';
 
-const config = {
+const configWithBranches = {
```

Comment:
May directly embed this config in `TestFixture` constructor to avoid intermediate variable.
When removing the variable I would suggest to add a comment describing the test case.